### DISCUSSION TITLE
Hide counter from cell

### DISCUSSION THREAD

**Discussion by capiet** [microsoft/vscode-jupyter#11603] [Questions and Answers] [answered]
Hello. I would like to know if it's possible to get rid out of the counter number on left of each cell in a notebook. The only way to change is restarting kernel. 
![image](https://user-images.githubusercontent.com/79054211/194946771-f6fb5200-f6aa-4663-a0b5-d7aab2d42804.png)

**Comment by DonJayamanne** [comment, marked as answer]
This contains the execution count (id) returned by the Jupyter kernel.
Generally tends to be an auto incrementing number that would tell you in what order a particular cell ran.

Currently its not possible to hide this. When ever you run a cell this will get added to the cell and will get displayed, and will also get saved in the iipynb file.

Out of curiosity why would you like to hide this?

**Reply by capiet** [reply]
Thanks for your reply. I often debug my code and this counter incrementing every time distracts me a little. Before that I thought that closing that label is that easy as hiding the execution time with settings.
